packages/squiggle-lang/__tests__/library/plot_test.ts

```diff
@@ -43,15 +46,14 @@ describe("Plot", () => {
         fn: {|x| x * 5},
         xPoints: [10,20,40]
       })`,
-      "Plot for numeric function",
-      true
+      "Plot for numeric function"
     );
 
     testEvalToMatch(
       `Plot.numericFn({|x,y| x * 5})`,
       `Error(Error: There are function matches for Plot.numericFn(), but with different arguments:
-  Plot.numericFn(fn: (Number) => Number, params?: {xScale?: Scale, yScale?: Scale, title?: String, xPoints?: List(Number)}) => Plot
-Was given arguments: ((x,y) => internal code)`
+  Plot.numericFn(fn: (Number) => Number, params?: {xScale?: Scale, yScale?: Scale, xPoints?: List(Number)}) => Plot
+Was given arguments: ((x: any, y: any) => Duration|Number|Dist))`
```

Comment:
Note how lambdas are now serialized to their inferred signatures.

(limitation: parameter annotations don't affect parameter types yet)